processors/pybossa_indications/processor.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from itertools import groupby
+
+import os
+import time
+import requests
+import pbclient as pbc
+import logging
+logger = logging.getLogger(__name__)
+
+PYBOSSA_URL = os.environ['PYBOSSA_URL']
+PYBOSSA_API_KEY = os.environ['PYBOSSA_API_KEY']
+PROJECT_ID = os.environ['PYBOSSA_PROJECT_INDICATIONS']
+
+
+def process(conf, conn):
+
+    # Get stuff from the DB
+    query = """
+        SELECT documents.id AS meta_id,
+            fda_approvals.id,
+            documents.name,
+            files.url,
+            fda_approvals.type,
+            fda_approvals.notes,
+            fda_approvals.supplement_number
+        FROM documents
+        INNER JOIN fda_approvals ON documents.fda_approval_id = fda_approvals.id
+        INNER JOIN files ON documents.file_id = files.id
+        WHERE documents.url IS NOT NULL
+        AND fda_approvals.type = 'New or Modified Indication'
+        GROUP BY fda_approvals.id,
+                documents.id,
+                files.id
+        ORDER BY fda_approvals.id
+    """
+
+    rows = list(conn['database'].query(query))
+    if rows:
+        _create_tasks(rows)
+    else:
+        logger.info('No new and modified indications found for tasks creation')
+
+
+def _create_tasks(rows):
+    tasks = []
+    for key, group in groupby(rows, lambda x: x['id']):
+        group = list(group)
+        first = group[0]
+        print(str(first['meta_id']))
+        task = {
+            'supplement_number': str(first['supplement_number']),
+            'approval_type': first['type'],
+            'approval_id': str(first['id']),
+            'meta_id': str(first['meta_id']),
+            'notes': first.get('notes', '')
```

Comment:
Do we use things like `supplement_number`, `approval_type` and others in the interface? If not, I would rather not add them here (and don't get them from the SQL query). As far as I can see, the `fda_approval_id` should be enough.

Also, if you're using a specific ID (e.g. `fda_approval_id`), name it as such and not a general `meta_id`, as it serves as a form of documentation on what ID it's about.